src/Execute/executeQuickInput.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+import {globalBackendMap} from '../Backend/Backend';
+import { Backend } from '../Backend/API';
+import { MultiStepInput } from '../Utils/MultiStepInput';
+import { exec } from 'child_process';
+import { appendFileSync } from 'fs';
+
+export async function runInferenceQuickInput(context: vscode.ExtensionContext) {
+  interface State {
+    backend: Backend;
+    modelPath: vscode.Uri;
+    inputSpec: string;
+    infer: vscode.QuickPickItem;
+  }
+
+  async function collectInputs() {
+    const state = {} as Partial<State>;
+    await MultiStepInput.run(input => pickBackend(input, state));
+    return state as State;
+  }
+
+  const title = 'Choose Executor Toolchain';
```

Comment:
How about `Set parameters to run inference`?

Reply:
I changed the titles different for each steps. :D